src/internal/packager/helm/zarf.go

```diff
@@ -49,32 +49,19 @@ func (h *Helm) UpdateZarfRegistryValues() error {
 }
 
 // UpdateZarfGiteaValues updates the Zarf git server deployment with the new state values
-func (h *Helm) UpdateZarfGiteaValues() error {
-	giteaValues := map[string]interface{}{
-		"gitea": map[string]interface{}{
-			"admin": map[string]interface{}{
-				"username": h.cfg.State.GitServer.PushUsername,
-				"password": h.cfg.State.GitServer.PushPassword,
-			},
-		},
-	}
-
-	h.chart = types.ZarfChart{
-		Namespace:   "zarf",
-		ReleaseName: "zarf-gitea",
-	}
+func (h *Helm) UpdateZarfGiteaValues(oldState *types.ZarfState) error {
+	oG := oldState.GitServer
+	nG := git.New(h.cfg.State.GitServer)
```

Comment:
nG / oG should ideally be the same type if they are going to share a similar name, oG is a GitServer struct but nG is a git client.

Reply:
mmm good point.